### ISSUE TITLE
Filter based on IDN and DNSSEC status

### ISSUE THREAD

**Issue by SaptakS** [SaraVieira/tld.party#9]
I have been extensively using tld.party to find out interesting TLDs and domains (I guess that tells a lot about my pass times). Something I have often faced is after finding an interesting TLD for a particular domain name, I find that IDN or DNSSEC doesn't exist, making it not possible for registration.

So, I feel a filter or a search of the form `IDN:exist <searchQuery>` would be useful. If this sounds like a good idea, I am happy to send a PR.

**Comment by SaraVieira** [comment]
Hey!

That would be a great idea but maybe just a checkbox would do the trick no?

**Comment by SaptakS** [comment]
Yep, a checkbox would do the trick. I was just suggesting something which would not require any additional Frontend component. But a checkbox would definitely be a better UX. Can I send a PR for the same?

**Comment by SaraVieira** [comment]
Hey!


Yeah, that would be amazing :)
